packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -1657,15 +1657,15 @@ test('mutable liquidity triggers and interest', async t => {
   await waitForPromisesToSettle();
   aliceUpdate = await E(aliceNotifier).getUpdateSince(aliceUpdate.updateCount);
   bobUpdate = await E(bobNotifier).getUpdateSince();
-  t.truthy(aliceUpdate.value.liquidated);
+  t.is(aliceUpdate.value.vaultState, 'liquidated');
```

Comment:
Agree it's better to use a constant instead of the literal. I'll define the constant in this test so that if the actual string VaultFactory returns changes this test will break.